devtools/azure-pipelines-linux.yml

```
@@ -7,10 +7,10 @@ jobs:
   strategy:
     matrix:
       Python36:
```

small nitpick, but if you're changings the name on line 12, you might as well change this to `Python37:`
```suggestion
       Python37:
```